### ISSUE TITLE
Add a stage describing how to extract a statement from literature, convert into reaction in PV and add literature reference

### ISSUE THREAD

**Issue by khanspers** [wikipathways/academy#32]
Suggestion from Curators meeting (Freddie):

[paraphrased]
"The Academy doesn't describe explicitly how to transcribe a statement (sentence) in a paper or other source into a reaction in WikiPathways. This should be described in detail in either the first step of the blue path, or a new step". 
Comment by KH: The language in subsequent steps describing modeling should be updated appropriately. 

https://academy.wikipathways.org/path.html

**Comment by khanspers** [comment]
Content was added to the [first conversion step](https://academy.wikipathways.org/stages/draw-catalysis/index.html):

- Referenced a paper with a sentence describing the reaction
- Quoted the sentence
- Described which parts of the sentence correspond to which parts of the reaction in the pathway model
- Updated PathVisio instructions accordingly

@fehrhart : Please review and see what you think.

**Comment by danidi** [comment]
Great explanation, thank you! 
Just a few questions: Shouldn't it be a gene product or protein instead of gene for CYP11A1? And what is meant with the "jnlp downloaded in this challenge"?

**Comment by fehrhart** [comment]
Looks good to me!
I would just recommend to call CYP a geneproduct or a protein data node according to PathVisio nomenclature. We dont have "enzyme data nodes" there.

Maybe an additional sentence that this also applies to protein modification reactions? There you would have 2 protein/gene product nodes instead of metabolites and add a state.

**Comment by khanspers** [comment]
@danidi : Thanks! I updated the instructions to state GeneProduct node, and will update that in other steps as well. The sentence about jnlp was left behind from when users would download a jnlp that launches PathVisio as Java Web Start. I deleted that sentence.
@fehrhart : I updated the list of objects extracted from the sentence to use PathVisio specific terms instead of general terms. Im still thinking of how to add the extra sentence. There are other examples as well, so maybe its better to adapt this and include in other steps that are relevant.
